### PR TITLE
Lumin Platform Support

### PR DESCRIPTION
**Pull request by StephenHodgson**
Overview
---
**ML Unity package & MLTP-8 Editor required & not included.**

- Adds Lumin Platform Support
    - Updates camera settings
    - Adds controller Support

### CODE REVIEW

File: ProjectSettings/ProjectSettings.asset [hunk 1]
```diff
@@ -138,6 +138,12 @@ PlayerSettings:
     hololens:
       depthFormat: 1
       depthBufferSharingEnabled: 0
+    lumin:
```
Comment (StephenHodgson) [inline ProjectSettings/ProjectSettings.asset]: This file needs to be reverted.

File: Assets/MixedRealityToolkit/_Core/Utilities/Editor/USB/USBDeviceListener.cs [hunk 2]
```diff
@@ -27,10 +27,7 @@ static USBDeviceListener()
 
         private static void NotifyUsbDevicesChanged(UsbDevice[] devices)
         {
-            if (UsbDevicesChanged != null)
-            {
-                UsbDevicesChanged.Invoke(devices);
-            }
+            UsbDevicesChanged?.Invoke(devices);
```
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit/_Core/Utilities/Editor/USB/USBDeviceListener.cs]: another superfluous change that can get reverted.

File: Assets/MixedRealityToolkit/_Core/Devices/OpenVR/OpenVRDeviceManager.cs [hunk 3]
```diff
@@ -14,7 +13,7 @@ namespace Microsoft.MixedReality.Toolkit.Core.Devices.OpenVR
     /// <summary>
     /// Manages Open VR Devices using unity's input system.
     /// </summary>
-    public class OpenVRDeviceManager : UnityJoystickManager, IMixedRealityComponent
+    public class OpenVRDeviceManager : UnityJoystickManager
```
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit/_Core/Devices/OpenVR/OpenVRDeviceManager.cs]: also not an important change

File: Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Pointers/BaseControllerPointer.cs [hunk 4]
```diff
@@ -190,11 +190,20 @@ public uint PointerId
             }
         }
 
+        private string pointerName = string.Empty;
```
[the remark this answers is not in the document]
Comment (StephenHodgson) [reply Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Pointers/BaseControllerPointer.cs]: I'm intending to do so.

File: Assets/MixedRealityToolkit/_Core/Utilities/Editor/USB/USBDeviceListener.cs (hunk 2 above)
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit/_Core/Utilities/Editor/USB/USBDeviceListener.cs]:
```suggestion
            if (UsbDevicesChanged != null)
            {
                UsbDevicesChanged.Invoke(devices);
            }
```

**Comment by SimonDarksideJ** [comment]
Can we rebase this in to a new PR in a new feature branch for testing.  With all the non-essential changes skipped? @StephenHodgson

**Comment by StephenHodgson** [comment]
Rebasing is bad mmkay?

But yes I can re-target it to a feature branch.

**Comment by SimonDarksideJ** [comment]
Ok, but what's your plan to exclude the stuff you added that is outside the scope of the PR (without deleting files in the dev branch) :D 
Or are you doing selective reversion :S

**Pull request merged**